src/main/java/backend/Team.java

```diff
@@ -0,0 +1,318 @@
+package backend;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Team {
+    private String name;
+    private String manager;
+    private int budget;
+    private int pointsAllTime;
+    private int pointsThisSeason;
+    private int winAllTime;
+    private int winThisSeason;
+    private Engine engine;
+    private Aerodynamicist aerodynamicist;
+    private Mechanic mechanic;
+    private Strategist strategist;
+    private List<Driver> drivers;
+
+    /**
+     * Create a new team.
+     * Create a new arraylist for drivers.
+     *
+     * @param name the name of the team
+     * @param manager the name of the manager
+     * @param budget the current budget of this team
+     * @param pointsAllTime the total amount of points until now
+     * @param pointsThisSeason the total amount of in this season
+     * @param winAllTime the total amount of wins until now
+     * @param winThisSeason the total wins in this season
+     * @param engine the engine of this team
+     * @param aerodynamicist the aerodynamicist of this team
+     * @param mechanic the mechanis of the team
+     * @param strategist the strategist of the team
+     */
+    public Team(String name, String manager, int budget, int pointsAllTime, int pointsThisSeason,
+                int winAllTime, int winThisSeason, Engine engine,
+                Aerodynamicist aerodynamicist, Mechanic mechanic, Strategist strategist) {
+        this.name = name;
+        this.manager = manager;
+        this.budget = budget;
+        this.pointsAllTime = pointsAllTime;
+        this.pointsThisSeason = pointsThisSeason;
+        this.winAllTime = winAllTime;
+        this.winThisSeason = winThisSeason;
```

Comment:
pointsAlltime, pointsThisSeason, winAllTime and winThisSeason can be removed for the constructor